lang/csharp/src/apache/codegen/Avro.codegen.csproj

```diff
@@ -1,166 +1,24 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<!--

```

Comment:
why are the license headers removed in this file and others?

Reply:
A mistake, glad you caught that one.